modules/serverless-kafka-broker-sasl-custom-config.adoc

```diff
@@ -0,0 +1,82 @@
+// Module is included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kafka.adoc
+
+[id="serverless-kafka-broker-sasl-custom-config_{context}"]
+= Configuring SASL authentication for the Kafka Broker
+
+.Prerequisites
+
+* A username and password for the Kafka cluster.
+* Choose the SASL mechanism to use, for example `PLAIN`, `SCRAM-SHA-256`, or `SCRAM-SHA-512`.
+* If TLS is enabled, you also need the `ca.crt` certificate file for the Kafka cluster.
+
+[NOTE]
+====
+Red Hat recommends to enable TLS in addition to SASL.
+====
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+[source,terminal]
+----
+$ oc create secret --namespace <namespace> generic <kafka_auth_secret> \
+  --from-literal=protocol=SASL_SSL \
+  --from-literal=sasl.mechanism=<sasl_mechanism> \
+  --from-file=ca.crt=caroot.pem \
+  --from-literal=password="SecretPassword" \
+  --from-literal=user="my-sasl-user"
+----
++
+[IMPORTANT]
+====
+Use the key names `ca.crt`, `password`, and `sasl.mechanism`. Do not change them.
+====
+
+. Create a `ConfigMap` for configuration as a YAML file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-kafka-broker-config
+data:
+  default.topic.partitions: <partitions>
+  default.topic.replication.factor: <replication_factor>
+  bootstrap.servers: <bootstrap_servers>
+  auth.secret.ref.name: <kafka_auth_secret>
+----
+
+. Apply the `ConfigMap` YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
```

Comment:
```suggestion
. Apply the config map:
+
[source,terminal]
----
$ oc apply -f <filename>
----
```

I'll leave it up to you how you want to format these. I'd probably go with "config map" as shown, but there are other suggestions earlier if you want to use `ConfigMap` either.